server/src/instant/scripts/newsletter.clj

```diff
@@ -0,0 +1,156 @@
+(ns instant.scripts.newsletter
+  "Send newsletters to our users
+
+  Usage: First run through these steps to prepare the email content
+    1. Create email content in Dropbox. Get sign off on the copy.
+    2. Export content as markdown and save to `www/intern/_emails/markdown/[slug].md`
+    3. Preview the styled markdown at `localhost:3000/intern/emails/[slug]`
+    4. Once preview looks good, generate html and text versions of the email
+       from `client/Makefile` via `make build-email`
+       Notes:
+        * We want both html and text to maximize delivery
+        * We use pandoc to convert markdown to html. If you don't have it,
+          you can install it via `brew install pandoc`
+        * If need be, you can edit the ouputs in`client/www/_emails/html/[slug].html`
+          and `client/www/_emails/txt/[slug].txt`
+    5. Refresh the page at `localhost:3000/intern/emails/[slug]`
+       and verify text version looks good via the text view button
+
+    And now go through the comment block at the bottom of this file to send
+    out the newsletter. Huzzah!
+  "
+
+  (:require
+   [instant.config :as config]
+   [clj-http.client :as clj-http]
+   [clojure.java.io :as io]
+   [clojure.string :as str]
+   [instant.util.json :refer [->json]]
+   [instant.util.tracer :as tracer]
+   [clojure.tools.logging :as log]))
+
+(def base-path (io/file (.getParent (io/file *file*)) "../../../../"))
+(def emails-path "server/resources/emails/emails.txt")
+(defn html-path [slug] (str "client/www/_emails/html/" slug ".html"))
+(defn txt-path [slug] (str "client/www/_emails/txt/" slug ".txt"))
+
+(defonce once (atom false))
+(defonce errors (atom []))
+(comment
+  (reset! once false)
+  (reset! errors []))
+
+(defn read-file
+  "Reads a file relative to a base path"
+  [base-path file-path]
+  (try
+    (let [full-path (io/file base-path file-path)]
+      (slurp full-path))
+    (catch Exception _
```

Comment:
nit: are you sure you want to add a try around this? Likely if the file doesn't exist, it'd be nice for the code execution to break out anyways

Reply:
Agreed! Pushed up an update!